### PR TITLE
W-13775249-user-permissions-step-missing

### PR DESCRIPTION
**Pull request by sarathecoubian**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/users.adoc [hunk 1]
```diff
@@ -75,6 +75,7 @@ include::partial$include-nav-steps-org-admin.adoc[]
 
 include::partial$include-nav-users.adoc[]
 
+. Click the name of the user to whom you want to assign permissions.
```
Comment (hannanelson) [inline modules/ROOT/pages/users.adoc]:
```suggestion
. Click the name of the user to assign permissions to.
```

File: modules/ROOT/pages/users.adoc [hunk 2]
```diff
@@ -75,6 +75,7 @@ include::partial$include-nav-steps-org-admin.adoc[]
...
 . In the *Permissions* section, click *Add permissions*.
```
Comment (hannanelson) [inline modules/ROOT/pages/users.adoc:79]: You have to select the section, such as Anypoint Code Builder. And **Add permissions** is not there for me. You select them from Permission(s).
image

File: modules/ROOT/pages/users.adoc [hunk 3]
```diff
@@ -75,6 +75,7 @@ include::partial$include-nav-steps-org-admin.adoc[]
...
 . Next to each permission you want to grant the user, select the checkbox.
 . Click *Next*.
```
Comment (hannanelson) [inline modules/ROOT/pages/users.adoc:81]: I don't see this. It looks like you click the "blue plus icon" to add it.

File: modules/ROOT/pages/users.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/users.adoc:79]: Oh, these instructions are for after you switch to the new features. I missed that.

File: modules/ROOT/pages/users.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/users.adoc:81]: I'm seeing expandable menus and checkboxes.

File: modules/ROOT/pages/users.adoc [hunk 4]
```diff
@@ -75,6 +75,7 @@ include::partial$include-nav-steps-org-admin.adoc[]
 
 include::partial$include-nav-users.adoc[]
 
+. Click the name of the user to assign permissions to.
```
Comment (emileemooreVT) [inline modules/ROOT/pages/users.adoc]:
```suggestion
. Click the name of the user to which you want to assign permissions.
```

**Review by emileemooreVT — DISMISSED** [review]

**Review by emileemooreVT — APPROVED** [review]

**Pull request merged**